Update README to include forking instructions before cloning repository (#10286)
WordPress/wordpress-develop: README.md

#### To start the development environment for the first time

> Goto https://github.com/WordPress/wordpress-develop and fork the repository to your own GitHub account. 

Reviewer: ```suggestion
Go to https://github.com/WordPress/wordpress-develop and fork the repository to your own GitHub account. 
```

> Then clone the forked repository to your computer using `git clone https://github.com/<your-username>/wordpress-develop.git`.

Reviewer: ```suggestion
Then clone the forked repository to your computer using `git clone https://github.com/<your-username>/wordpress-develop.git`. Add the origin repo as an `upstream` remote via `git remote add upstream https://github.com/WordPress/wordpress-develop.git`; then you can keep your branches up to date via `git pull --ff upstream/trunk`, for example.
```

> 

Reviewer: ```suggestion

Alternatively, if you have the GitHub CLI installed, you can just run `gh repo fork WordPress/wordpress-develop --clone --remote`. This will automatically add an `upstream` remote as well.

```

#### To start the development environment for the first time

Go to https://github.com/WordPress/wordpress-develop and fork the repository to your own GitHub account. 

Then clone the forked repository to your computer using `git clone https://github.com/<your-username>/wordpress-develop.git`. Add the origin repo as an `upstream` remote via `git remote add upstream https://github.com/WordPress/wordpress-develop.git`; then you can keep your branches up to date via `git pull --ff upstream/trunk`, for example.

> Alternatively, if you have the GitHub CLI installed, you can just run `gh repo fork WordPress/wordpress-develop --clone --remote`. This will automatically add an `upstream` remote as well.

Reviewer: ```suggestion
Alternatively, if you have the [GitHub CLI](https://cli.github.com/) installed, you can just run `gh repo fork WordPress/wordpress-develop --clone --remote`. This will automatically add an `upstream` remote as well.
```

#### To start the development environment for the first time

You can get started using the local development environment with these steps:

1. Go to https://github.com/WordPress/wordpress-develop and fork the repository to your own GitHub account. 
1. Then clone the forked repository to your computer using `git clone https://github.com/<your-username>/wordpress-develop.git`.

> 1. Add the origin repo as an `upstream` remote via `git remote add upstream https://github.com/WordPress/wordpress-develop.git`;

Reviewer 2: ```suggestion
1. Navigate into the directory for the cloned repository using `cd wordpress-develop`
1. Add the origin repo as an `upstream` remote via `git remote add upstream https://github.com/WordPress/wordpress-develop.git`;
```

Missing step. I believe you'll just get a "fatal: not a git repository" error without it.

1. Then you can keep your branches up to date via `git pull --ff upstream/trunk`, for example.

> Alternatively, if you have the [GitHub CLI](https://cli.github.com/) installed, you can just run `gh repo fork WordPress/wordpress-develop --clone --remote`. This will automatically add an `upstream` remote as well.

Reviewer 2: ```suggestion
Alternatively, if you have the [GitHub CLI](https://cli.github.com/) installed, you can simply run `gh repo fork WordPress/wordpress-develop --clone --remote` ([docs](https://cli.github.com/manual/gh_repo_fork)). This command will:
1. Fork the repository to your account (use the `--org` flag to clone into an organization).
1. Clone the repository to your machine. 
1, Add `WordPress/wordpress-develop` as `upstream` and set it to the default `remote` repository

After this, remember to run `cd wordpress-develop`.
```

> 
> After either of the two options above, move to the repository folder `cd wordpress-develop` and run the following commands in a terminal:

Reviewer 2: ```suggestion
```

#### To start the development environment for the first time

You can get started using the local development environment with these steps:

1. Go to https://github.com/WordPress/wordpress-develop and fork the repository to your own GitHub account. 
1. Then clone the forked repository to your computer using `git clone https://github.com/<your-username>/wordpress-develop.git`.
1. Navigate into the directory for the cloned repository using `cd wordpress-develop`
1. Add the origin repo as an `upstream` remote via `git remote add upstream https://github.com/WordPress/wordpress-develop.git`;
1. Then you can keep your branches up to date via `git pull --ff upstream/trunk`, for example.

Alternatively, if you have the [GitHub CLI](https://cli.github.com/) installed, you can simply run `gh repo fork WordPress/wordpress-develop --clone --remote` ([docs](https://cli.github.com/manual/gh_repo_fork)). This command will:
1. Fork the repository to your account (use the `--org` flag to clone into an organization).
1. Clone the repository to your machine. 

> 1, Add `WordPress/wordpress-develop` as `upstream` and set it to the default `remote` repository

Reviewer 3: ```suggestion
1. Go to https://github.com/WordPress/wordpress-develop and fork the repository to your own GitHub account. 
1. Then clone the forked repository to your computer using `git clone https://github.com/<your-username>/wordpress-develop.git`.
1. Navigate into the directory for the cloned repository using `cd wordpress-develop`.
1. Add the origin repo as an `upstream` remote via `git remote add upstream https://github.com/WordPress/wordpress-develop.git`.
1. Then you can keep your branches up to date via `git pull --ff upstream/trunk`, for example.

Alternatively, if you have the [GitHub CLI](https://cli.github.com/) installed, you can simply run `gh repo fork WordPress/wordpress-develop --clone --remote` ([docs](https://cli.github.com/manual/gh_repo_fork)). This command will:
1. Fork the repository to your account (use the `--org` flag to clone into an organization).
1. Clone the repository to your machine. 
1. Add `WordPress/wordpress-develop` as `upstream` and set it to the default `remote` repository
```
